app/_components/cbio-util.js

```diff
@@ -285,10 +285,10 @@ cbio.util = (function() {
 
     // check for IE 11
     if (!(browser.msie ||
-      browser.firefox ||
-      browser.chrome ||
-      browser.safari ||
-      browser.opera)) {
+        browser.firefox ||
```

Comment:
Is this file going to be updated in cbioportal or are these updates from cbioportal?

Reply:
This is from portal.